package.json

```diff
@@ -19,11 +19,15 @@
     "@astrojs/db": "^0.11.6",
     "@astrojs/netlify": "^5.3.2",
     "astro": "^4.11.0",
+    "crypto": "^1.0.1",
+    "crypto-js": "^4.2.0",
+    "pg": "^8.12.0",
     "typescript": "^5.4.5"
   },
   "devDependencies": {
     "prettier": "^3.3.2",
     "prettier-plugin-astro": "^0.14.0",
+    "prisma": "^5.16.1",
```

Comment:
I had not fully process when you mentioned using Prisma for the work you. AstroDB has its own ORM (wrapper for drizzle) and the database we are using is a SQLite one hosted in Astro Studio. The Postgres library won't be needed either.